src/plugins/pdfPlayer/plugin.js

```diff
@@ -279,16 +279,21 @@ export class PdfPlayer {
 
     renderPage(canvas, number) {
         this.book.getPage(number).then(page => {
-            const original = page.getViewport({ scale: 1 });
+            const width = dom.getWindowSize().innerWidth;
+            const height = dom.getWindowSize().innerHeight;
+            const viewport = page.getViewport({ scale: 5 });
```

Comment:
I would recommend trying this instead of picking an arbitrary value for the scale.

```suggestion
            const scale = Math.ceil(window.devicePixelRatio || 1);
            const viewport = page.getViewport({ scale });
```